test/src/webgl/context-features.spec.js

```diff
@@ -78,49 +78,50 @@ test('webgl#caps#hasFeatures(WebGL2)', t => {
   t.end();
 });
 
-/*
-NOTE: Disabling the test as it crashes when tested by 'npm run test-brwoser'
- when it crashes it also leave global state modified, causing failure in other unit tests
 test('webgl#caps#canCompileGLGSExtension', t => {
   const {gl} = fixture;
-  const oldNavigator = window.navigator;
+
+  const userAgentNonIE = 'Mozilla/5.0 (Macintosh; Intel Mac OS X 10_13_3) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/65.0.3325.162 Safari/537.36';
+  const userAgentOldIE = 'Mozilla/5.0 (Windows NT 6.3; Trident/7.0; rv:11.0) like Gecko';
 
   t.ok(typeof canCompileGLGSExtension === 'function', 'canCompileGLGSExtension defined');
 
-  // Non-IE version.
+  // // Non-IE version.
   const getShaderParameterStub = sinon.stub(gl, 'getShaderParameter');
```

Comment:
Add comment explain why we are checking whether this particular function was called?

Reply:
Updated the test assertion comments to be more descriptive.